src/schema/sale_artwork.ts

```diff
@@ -364,4 +382,24 @@ const SaleArtwork: GraphQLFieldConfig<void, ResolverContext> = {
   },
 }
 
+const shouldFetchArtwork = (
+  fieldNode: Readonly<FieldNode>,
+  fragments: any
+): boolean => {
+  const fetchArtworkFields = ["edition_of", "edition_sets"]
```

Comment:
for now only adding these two fields as fields triggering extra call, we can add more fields when needed.

Reply:
(Off-topic If we had gravitype up and running we could reflect on the different json versions and know that way if shallow is enough or not.)